src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -889,14 +553,14 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
               <Skeleton isLoaded={!quoteLoading}>
                 <Amount.Fiat
                   fontWeight='bold'
-                  value={bnOrZero(networkFeeCryptoBaseUnit)
+                  value={bnOrZero(estimatedFeesData?.estimatedFees.fast.txFee)
                     .div(bn(10).pow(feeAsset.precision))
                     .times(feeMarketData.price)
                     .toFixed()}
                 />
                 <Amount.Crypto
                   color='text.subtle'
-                  value={bnOrZero(networkFeeCryptoBaseUnit)
+                  value={bnOrZero(estimatedFeesData?.estimatedFees.fast.txFee)
```

Comment:
can also use `fromBaseUnit` helper here and above

Reply:
and gm c3937b3de53e37821bf9f9bec7ebd6c5f15df04b